src/main/java/org/jabref/gui/fieldeditors/ContextMenuAddable.java

```diff
@@ -5,11 +5,13 @@
 
 import javafx.scene.control.MenuItem;
 
+import org.jabref.gui.keyboard.KeyBindingRepository;
+
 public interface ContextMenuAddable {
     /**
      * Adds the given list of menu items to the context menu. The usage of {@link Supplier} prevents that the menus need
      * to be instantiated at this point. They are populated when the user needs them which prevents many unnecessary
      * allocations when the main table is just scrolled with the entry editor open.
      */
-    void initContextMenu(final Supplier<List<MenuItem>> items);
+    void initContextMenu(final Supplier<List<MenuItem>> items, KeyBindingRepository keyBindingRepository);
```

Comment:
I wonder why this is necessary? Can't the implementation classes use the `Injector` to get an instance?

Reply:
Yes, but afterburner is not a di framework. Using the Injector class is still only an iteration step towards a proper di framework, so i decided for constructor injection here.